soperator/modules/available_resources/main.tf

```diff
@@ -251,5 +251,5 @@ locals {
 }
 
 data "units_data_size" "k8s_ephemeral_storage_reserve" {
-  gibibytes = 64
+  gibibytes = 48
```

Comment:
I suggest to set this to 64, just in case

Reply:
Done